src/components/HURUmap/Panel/useStyles.js

```diff
@@ -0,0 +1,32 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(({ typography, palette }) => ({
+  root: {},
+  paper: {
+    background: "transparent",
+    border: "none",
+    visibility: "visible !important",
```

Comment:
by default, the paper is invisible (style is on the element itself, not a class) when closed, and since we want the panel buttons to be visible all the time, I had to override it. @kilemensi

Reply:
Didn't the [mini variant drawer](https://v4.mui.com/components/drawers/#mini-variant-drawer) work?